packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -1719,7 +1719,7 @@ export class ProjectService extends BaseService {
                         );
                     }
 
-                    await this.analytics.track({
```

Comment:
Hmm, should these ever be awaited? I see a few other places we are doing this.

Reply:
no, its not a promise function and is redundant